addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties

```diff
@@ -4,8 +4,16 @@ paramminer.error.nofile = File not found : {0}
 paramminer.panel.msg = <html>Param Miner panel</html>
 paramminer.panel.title = Param Miner
 paramminer.popup.msg = Example right click menu invoked for:\n {0}
-paramminer.popup.title = Param Miner: Example right click menu
+paramminer.popup.title = Param Miner
 paramminer.topmenu.tools.msg = Param Miner: Useful for finding cache poisoning vulnerabilities
 paramminer.topmenu.tools.title = Param Miner
 paramminer.panel.tab.history = History
-paramminer.panel.tab.output = Output
\ No newline at end of file
+paramminer.panel.tab.output = Output
+paramminer.dialog.url = Url:
```

Comment:
I'd suggest `URL:`

Reply:
I kept it Url because it was looking odd :). My first commit had it as URL :).